Parsers/ASimAuthentication/Tests/PaloAlto_CortexDataLake_ASimAuthentication_SchemaTest.csv

```diff
@@ -0,0 +1,84 @@
+Result
+"(1) Warning: Missing recommended field [DvcAction]"
+"(1) Warning: Missing recommended field [EventResultDetails]"
+"(1) Warning: Missing recommended field [TargetHostname]"
+"(2) Info: Missing optional alias [Application] aliasing non-existent column [TargetAppName]"
+"(2) Info: Missing optional field [ActingAppId]"
+"(2) Info: Missing optional field [ActingAppName]"
+"(2) Info: Missing optional field [ActingAppType]"
+"(2) Info: Missing optional field [ActorOriginalUserType]"
+"(2) Info: Missing optional field [ActorScopeId]"
+"(2) Info: Missing optional field [ActorScope]"
+"(2) Info: Missing optional field [ActorSessionId]"
+"(2) Info: Missing optional field [ActorUserId]"
+"(2) Info: Missing optional field [ActorUserType]"
+"(2) Info: Missing optional field [ActorUsername]"
+"(2) Info: Missing optional field [AdditionalFields]"
+"(2) Info: Missing optional field [DvcDescription]"
+"(2) Info: Missing optional field [DvcInterface]"
+"(2) Info: Missing optional field [DvcMacAddr]"
+"(2) Info: Missing optional field [DvcOriginalAction]"
+"(2) Info: Missing optional field [DvcOsVersion]"
+"(2) Info: Missing optional field [DvcOs]"
+"(2) Info: Missing optional field [DvcScopeId]"
+"(2) Info: Missing optional field [DvcScope]"
+"(2) Info: Missing optional field [DvcZone]"
+"(2) Info: Missing optional field [EventOriginalSubType]"
+"(2) Info: Missing optional field [EventOwner]"
+"(2) Info: Missing optional field [EventReportUrl]"
+"(2) Info: Missing optional field [EventSubType]"
+"(2) Info: Missing optional field [HttpUserAgent]"
+"(2) Info: Missing optional field [LogonMethod]"
+"(2) Info: Missing optional field [LogonTarget]"
+"(2) Info: Missing optional field [RuleName]"
+"(2) Info: Missing optional field [RuleNumber]"
+"(2) Info: Missing optional field [Rule]"
+"(2) Info: Missing optional field [SrcDescription]"
+"(2) Info: Missing optional field [SrcDeviceType]"
+"(2) Info: Missing optional field [SrcDvcId]"
+"(2) Info: Missing optional field [SrcDvcScopeId]"
+"(2) Info: Missing optional field [SrcDvcScope]"
+"(2) Info: Missing optional field [SrcGeoCity]"
+"(2) Info: Missing optional field [SrcGeoCountry]"
+"(2) Info: Missing optional field [SrcGeoLatitude]"
+"(2) Info: Missing optional field [SrcGeoLongitude]"
+"(2) Info: Missing optional field [SrcGeoRegion]"
+"(2) Info: Missing optional field [SrcIsp]"
+"(2) Info: Missing optional field [SrcOriginalRiskLevel]"
+"(2) Info: Missing optional field [SrcPortNumber]"
+"(2) Info: Missing optional field [SrcRiskLevel]"
+"(2) Info: Missing optional field [TargetAppId]"
+"(2) Info: Missing optional field [TargetAppName]"
```

Comment:
@niralishah-crest 
These Auth logs are for which application? Where are users authenticating in? Palo Alto Data Lake App itself?

Reply:
haven't seen app info in sample log.